### PR TITLE
Custom response

### PR DESCRIPTION
**Pull request by AvihayMReblaze**

### CODE REVIEW

File: src/doc-editors/CustomResponseEditor.vue [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+        }, {} as CustomResponse['params']['headers'])
+        this.emitDocUpdate()
+      },
+      deep: true,
+    },
+
+  },
+  props: {
+    selectedDoc: Object,
+    apiPath: String,
```
Comment (Aviv-Galmidi) [inline src/doc-editors/CustomResponseEditor.vue:185]: add the `apiPath` to the editor - look at ACLEditor (or any other doc editor) to understand how it's being used

File: src/doc-editors/CustomResponseEditor.vue [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+        status: null,
+        headers: {},
+        content: '',
+      }
+      this.emitDocUpdate()
+    },
+  },
+})
+</script>
+<style scoped lang="scss">
```
Comment (Aviv-Galmidi) [inline src/doc-editors/CustomResponseEditor.vue:252]: Remove unused classes from the style

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]

File: src/doc-editors/CustomResponseEditor.vue (hunk 2 above)
Comment (AvihayMReblaze) [reply src/doc-editors/CustomResponseEditor.vue:252]: done

File: src/doc-editors/CustomResponseEditor.vue (hunk 1 above)
Comment (AvihayMReblaze) [reply src/doc-editors/CustomResponseEditor.vue:185]: done

File: src/doc-editors/CustomResponseEditor.vue [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+.status-code {
```
Comment (Aviv-Galmidi) [inline src/doc-editors/CustomResponseEditor.vue:253]: Do we still need the `margin-right` ? Might be something that was used before we switched to columns

Comment (AvihayMReblaze) [reply src/doc-editors/CustomResponseEditor.vue:253]: No, we don't need. I removed it.

**Review by Aviv-Galmidi — APPROVED** [review]

**Pull request merged**